docs/en/FAQ/Compatible-with-other-javaagent-bytecode-processing.md

```diff
@@ -0,0 +1,53 @@
+## Compatible with other javaagent bytecode processing
+
+### Problem
+1. when use skywalking agent, arthas can't work well 
+https://github.com/apache/skywalking/pull/4858
+
+2. Custom java agent retransform class fails with Skywalking agent  
+ https://github.com/hengyunabc/skywalking-error-demo
```

Comment:
```suggestion
2. Java agent retransforming class fails with Skywalking agent, such as in this [demo](https://github.com/hengyunabc/skywalking-error-demo)
```

Also, this code, would you like to push them into [SkyAPMTest](https://github.com/SkyAPMTest), I could create a repo for you and invite you in. The reason I recommend there is, we host unofficial tests and demo codes there, which need to be linked from the official repo.

Reply:
Yes, please create a repo: skywalking-retransform-demo

Reply:
Repo  https://github.com/SkyAPMTest/retransform-conflict-demo created, and sent the invitation to you.